Manual.adoc

```diff
@@ -1,43 +1,57 @@
+////
+TODO:
+
+- A section on automabatable params
+- Use "preset" instead of "patch", to match the UI.
+////
+
 = Zyn-Fusion User Manual
 :icons: font
 
+// Sets the ability to treat chapter text as links
+:sectlinks:
+
+// Enables section numbers
+:sectnums:
+:sectnumlevels: 4
+
 == Introduction
 include::Introduction.adoc[]
 
-== Basic Topics
-
-=== User Interface
+== User Interface
 include::UserInterface.adoc[]
 
-=== Synthesizer Modules
-include::SynthesizerModules.adoc[]
+// === Synthesizer Modules
+// include::SynthesizerModules.adoc[]
```

Comment:
I'm guessing that commenting these out is part of the WIP nature of this PR?

Reply:
Yes, as I finish each part up, I uncomment the next one. Makes the PDF and TOC more managable, since the WIP content is always at the bottom and not somewhere in the middle of the doc.